alpaca/data/models/corporate_actions.py

```diff
@@ -0,0 +1,99 @@
+from datetime import date
+from typing import Dict, List, Optional
+
+from alpaca.common.models import ValidateBaseModel as BaseModel
+from alpaca.common.types import RawData
+from alpaca.data.models.base import BaseDataSet, TimeSeriesMixin
+
+
+class CorporateAction(BaseModel):
```

Comment:
This is the "union" of all possible corporate actions? I really don't like this approach. Either create a model for all corporate action types (reverse split, cash dividend, etc.), or simply make it completely dynamic, either by using a dict type, or automatically generating the fields from the returned objects.

Reply:
Now I understand why you did it like this: to be able to make one big dataframe with all the corporate action types 🤔 I still don't like this, but if we want to achieve this, this may be the best solution.

Reply:
Please let me try some.